span-stacktrace/README.md

```diff
@@ -9,6 +9,37 @@ As a consequence it should only be used when the span duration is known, thus on
 However, the current SDK API does not allow to modify span attributes on span end, so we have to
 introduce other components to make it work as expected.
 
+## Usage
+
+This extension does not support autoconfiguration because it requires strict ordering of span
+processors to ensure it's executed between the last span processor and the span exporter.
```

Comment:
I'd rather word it as "needs to wrap the `SimpleSpanProcessor` or `BatchingSpanProcessor` which invokes the `SpanExporter`.